plugins/GSdx/GSRendererDX9.cpp

```diff
@@ -56,6 +56,36 @@ bool GSRendererDX9::CreateDevice(GSDevice* dev)
 	return true;
 }
 
+void GSRendererDX9::EmulateTextureShuffleAndFbmask()
+{
+	size_t count = m_vertex.next;
+	GSVertex* v = &m_vertex.buff[0];
+
+	// Shadow_of_memories_Shadow_Flickering (Okami mustn't call this code)
```

Comment:
Hum this code should be moved where we compute the value of m_texture_shuffle (likely in GSRendererHW). I don't like the copy-past for each renderers

Reply:
Which line ?
https://github.com/PCSX2/pcsx2/blob/master/plugins/GSdx/GSRendererHW.cpp#L882

Reply:
Yes below this line in RendererHW. Add the below block, because this block correct the detection of m_texture_shuffle. It should be done as soon as we detect the effect.